### PR TITLE
Add admin-level, common AJAX support

### LINKED ISSUE #1738: v156: Include base AJAX processing in the admin?
Is it time to update the admin to use the base AJAX processing that's included in the storefront?  

I'm thinking that an `/admin/ajax.php` would `require` the version in the storefront root, the admin header would require the storefront `/includes/template_default/jscript/jscript_framework.php` and the admin's `classes` directory would now include an `ajax` sub-directory ... mimicking the storefront behavior.

### PR DESCRIPTION
**Pull request by lat9**
Closes #1738

### CODE REVIEW

File: admin/includes/init_includes/init_admin_auth.php [hunk 1]
```diff
@@ -10,6 +10,22 @@
 
 define('SUPERUSER_PROFILE', 1);
 
+// -----
+// Special handling for AJAX requests.  Return a 'logged_out' error if no admin session
+// is active; otherwise, bypass the remainder of the authorization checks.
+//
+if (basename($PHP_SELF) != FILENAME_AJAX . '.php') {
```
Comment (drbyte) [inline admin/includes/init_includes/init_admin_auth.php]: Should this be `==` instead of `!=` ?

Comment (lat9) [reply admin/includes/init_includes/init_admin_auth.php]: It's not my month ... of course it should be.

**Comment by lat9** [comment]
That's most likely going to correct that json output that I saw on admin session-timeout, too.

**Comment by drbyte** [comment]
Thanks!

**Pull request merged**